### PR TITLE
FotoToon Activity Interactive Buttons

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Fixes issue #1221 

image

@llaske I've updated the toon.js file so that the interactive buttons appear above 
Could you please review it

### CODE REVIEW

**Comment by llaske** [comment]
Very cool! :+1: 

BTW the issue is still on the main title. Could you fix it too?

1H1pvPqLV1

**Comment by Vishalk91-4** [comment]
@llaske 
I've tried and updated it, it now looks like this, tried hard but that dotted rectangle wasn't going back , the things were becoming static on apply it.

image

**Comment by llaske** [comment]
> @llaske I've tried and updated it, it now looks like this, tried hard but that dotted rectangle wasn't going back , the things were becoming static on apply it.

It's because you've added controls before the dotted line was drawn.
Add it after, it should work.

File: activities/Fototoon.activity/js/toon.js [hunk 1]
```diff
@@ -1289,6 +1289,8 @@ define(["easel","sugar-web/datastore","sugar-web/env","webL10n","humane"], funct
         };
 
         this.createControls = function() {
+	    this._stage.addChild(this._resizeButton);
```
Comment (llaske) [inline activities/Fototoon.activity/js/toon.js:1292]: Warning: indentation is wrong on this line, it mix TAB and SPACE.

Comment (Vishalk91-4) [reply activities/Fototoon.activity/js/toon.js]: I've made the indentation proper now

Comment (llaske) [reply activities/Fototoon.activity/js/toon.js]: okay so move these lines after the drawing of dotted lines, it should work for title too

Comment (Vishalk91-4) [reply activities/Fototoon.activity/js/toon.js]: It didn't change on applying it to the back there

Comment (llaske) [reply activities/Fototoon.activity/js/toon.js]: Yes but if you want to fix the issue regarding main title, these lines should be moved.

**Comment by Vishalk91-4** [comment]
@llaske, now it's working perfectly as expected

image

**Comment by llaske** [comment]
@Vishalk91-4 could you ensure that only changed lines are updated.
Lot of lines change only for indentation reason:

image

**Comment by Vishalk91-4** [comment]
@llaske, Now I've removed the unnecessary indentations, now it's structured properly

**Comment by llaske** [comment]
> Now I've removed the unnecessary indentations, now it's structured properly

Good. It's important for maintenance reason to be able to identify only lines really changed by the PR.

Thanks.

**Comment by Vishalk91-4** [comment]
> > Now I've removed the unnecessary indentations, now it's structured properly
> 
> Good. It's important for maintenance reason to be able to identify only lines really changed by the PR.
> 
> Thanks.

Yes, I'll surely keep that in mind while working on other things

**Pull request merged**